docsite/docs/Reference/variables-and-functions.md

```diff
@@ -36,24 +36,47 @@ context.current_model.tests
 #= [CurrentTest(name='not_null', modelname='historical_ozone_levels, column='ds', status='Pass')]
 ```
 
-## `ref` and `source` functions
+## Read functions
 
-There are also available some familiar functions from `dbt`:
+There are also available some familiar functions from `dbt` to read the models and sources as a Pandas DataFrame.
```

Comment:
`ref` and `source` functions are available in fal scripts to read the models and sources as a Pandas DataFrame.